pkg/odo/cli/build_images/build_images.go

```diff
@@ -58,7 +58,7 @@ func (o *BuildImagesOptions) Validate() (err error) {
 }
 
 // Run contains the logic for the odo command
-func (o *BuildImagesOptions) Run() (err error) {
+func (o *BuildImagesOptions) Run(cmdline cmdline.Cmdline) (err error) {
```

Comment:
your are passing new argument, but  don't see it being used inside this function

Reply:
Eventually all commands will need to record telemetry and telemetry functions need context